code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -271,21 +272,35 @@
 	Proj.on_hit(src)
 	return 0
 
+/mob/living/simple_animal/proc/adjustHealth(amount)
+	if(status_flags & GODMODE)
+		return 0
+	bruteloss = Clamp(bruteloss + amount, 0, maxHealth)
+	updatehealth()
+	return amount	
+	
 /mob/living/simple_animal/adjustBruteLoss(amount)
-	if(!ignored_damage_types[BRUTE])
-		..()
+	if(damage_coeff[BRUTE])
+		. = adjustHealth(amount*damage_coeff[BRUTE])
```

Comment:
After further investigation neither myself nor Crystal can locate "adjustHealth" proc for living/simple_animal